### ISSUE TITLE
Feature Request: Textbox letter by letter reveal + letter sound effect

### ISSUE THREAD

**Issue by CronosZX** [ohrrpgce/ohrrpgce#1211]
Don't know if this has been requested already, seeing as how it is pretty much a staple of rpgs, but having the option to reveal textboxes letter by letter instead of line by line would be nice (there was a script for this somewhere but couldn't get it to work).

Likewise the added option to play a sound effect on each letter (with the preempt bit on so the sounds don't end up canceling each other) would help, because as of now the only way to achieve this effect is to manually prepare sound files of different lengths in advance (the line sound effect isn't very helpful when you only have like 3-4 lines per box and they show up almost instantly).

**Comment by rversteegen** [comment]
Yes, this is planned and in fact already largely implemented. I'll finish off my work on the new text and font system after hróðvitnir. Including the per-character sfx. Actually, I think we need some settings for "play sound every X characters", "loop instead of preempt", and eventually also for playing multiple instances of the each-char (or each-line) sound at once, which isn't currently supported by our audio backends.
